docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
   gateway-config:
     image: "nebraltd/hm-config:805f84ef025883d25b129229d7367c6f6c350722"
     environment:
-      - 'FIRMWARE_VERSION=2021.09.27.0'
+      - 'FIRMWARE_VERSION=2021.09.27.3'
```

Comment:
Should this be bumped to `2021.10.05.1`? The most recent change was done today.

Reply:
@marvinmarnold could do - however, i was basically bumping the last bit only as the helium firmware is still 2021.09.27.0 version.

Not sure what you think is best?

Reply:
Ah right. The way you have it makes the most sense!